site/gatsby-site/src/components/cite/NotifyButton.js

```diff
@@ -0,0 +1,33 @@
+import { faEnvelope } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React from 'react';
+const { Button, Spinner } = require('flowbite-react');
```

Comment:
Why are you using `require` instead of `import` not sure if it makes a difference, but I would try to keep it consistent.